client/src/components/ExploreGames/index.js

```diff
@@ -1,9 +1,8 @@
-import React from 'react'
-import GameCard from '../GameCard'
-import SearchGames from '../SearchGames'
-import bgaApiCall from '../../utils/bgaApiCall'
-import M from 'materialize-css'
```

Comment:
still need this.

Reply:
the materialize part that is

Reply:
nothing in this component calls materialize. much of what is needed is auto initiated in the app.js file, the parent most file.